docs/reference/components/sinks/http.cue

```diff
@@ -114,6 +114,16 @@ components: sinks: http: {
 				examples: ["https://10.22.212.22:9000/endpoint"]
 			}
 		}
+		healthcheck: type: object: options: uri: {
```

Comment:
Would this make more sense as an option to the global `healthcheck` in `sinks.cue`, with a flag to enable it like for `healthcheck.enabled`?

Reply:
It seams useful.

Reply:
Only one sink has this option, and there is some work to add this to cue, so let's wait for when it's used in at least two sinks.

Reply:
Agreed